filechooser-portal/FileChooserDialog.vala

```diff
@@ -473,6 +473,56 @@ public class Files.FileChooserDialog : Hdy.Window, Xdp.Request {
         }
     }
 
+    private void accept () {
+        if (action == Gtk.FileChooserAction.SAVE || action == Gtk.FileChooserAction.CREATE_FOLDER) {
+            var file = chooser.get_file ();
+            if (file == null) {
+                response (Gtk.ResponseType.CANCEL);
+                return;
+            }
+
+            if (file.query_exists ()) {
+                var display_name = file.get_basename ();
+                var primary = _("Replace “%s”?".printf (display_name));
```

Comment:
```suggestion
                var primary = _("Replace “%s”?").printf (display_name);
```
For the i18n to work you have to first translate the raw string then replace the placeholders